app/components/support_interface/state_explanation_component.html.erb

```diff
@@ -0,0 +1,46 @@
+<h2 class='govuk-heading-l' id='<%= state_name %>'><%= t "application_states.#{state_name}.name" %></h2>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-body">
+      <%= t "application_states.#{state_name}.description" %>
+    </div>
+
+    <% if state.events.any? %>
+      <h4 class='govuk-heading-m'>Actions</h4>
+
+      <ul class="govuk-list">
+      <% state.events.each do |_, events| %>
+        <% event = events.first # this is an array, but will always have a single item %>
```

Comment:
The whole section underneath has wonky syntax highlighting on GitHub due to this comment; nbd though